providers/implementations/keymgmt/dsa_kmgmt.c

```diff
@@ -192,12 +192,16 @@ static int dsa_export(void *keydata, int selection, OSSL_CALLBACK *param_cb,
                       void *cbarg)
 {
     DSA *dsa = keydata;
-    OSSL_PARAM_BLD *tmpl = OSSL_PARAM_BLD_new();
+    OSSL_PARAM_BLD *tmpl = NULL;
```

Comment:
Drop the `= NULL` here, it's not doing anything.

Reply:
This seems to have no real impact. I just keep the same form as `ec_export()` in `ec_kmgmt.c:429` and `dh_export()` in `dh_kmgmt.c:190` etc.